Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/XmlDocumentationShouldAddValueAnalyzer.cs

```diff
@@ -154,6 +160,57 @@ private void AnalyzeNamedNode(SyntaxNodeAnalysisContext context, Func<string> na
 			}
 		}
 
+		/// <summary>
+		/// Checks if there are any non-summary XML documentation elements that contain meaningful content.
+		/// Elements like param, returns, exception, etc. are considered if they have useful information.
+		/// </summary>
+		/// <param name="xmlElements">All XML documentation elements</param>
+		/// <param name="lowercaseName">The lowercase name of the documented element</param>
+		/// <returns>True if there are meaningful non-summary elements, false otherwise</returns>
+		private bool HasUsefulNonSummaryElements(IEnumerable<XmlElementSyntax> xmlElements, string lowercaseName)
+		{
+			var meaningfulElements = new[] { "param", "returns", "exception", "remarks", "example", "value" };
```

Comment:
Why don't we include `summary` here also, the code seems to be similar, right?

Reply:
The rule is now: If **any** of the tags contain useful information, we keep the entire comments.